### PR TITLE
docs: Add Vscode (#39512)

### PR DESCRIPTION
**Pull request by ajayrose25**

### CODE REVIEW

File: vscode.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# express-server
+
+ # Features of VS Code
```
Comment (VisaaliSuccessive) [inline vscode.md]: - remove extra space prior to the start of the heading

- make this as h2 tag

File: vscode.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+1.**A very helpful syntax highlighter and autocomplete feature that provides smart completions based on variable types, function definitions, and imported modules.**
+
+2.**The built in debugger helps you accelerate editing, compiling, and debugging by adding breakpoints and watchers. By default it comes with support for NodeJS and can debug anything that is transpiled to JavaScript but other runtimes like C++ or Python will require an extension to be installed.**
+
+3.**There is a built in GUI for Git for the most common commands which makes it really easy to instantly see the changes you're making in your project.**
+
+4.**VS Code offers icon theming so that you can switch between icon themes in the same manner you'd switch code themes.**
+
+5.**The ability to work with a full featured integrated terminal let's you run everything you normally would from your terminal of choice.**
```
Comment (VisaaliSuccessive) [inline vscode.md]: Unbold this, only headings must be in bold rest can be in normal font weight

File: vscode.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+2.**The built in debugger helps you accelerate editing, compiling, and debugging by adding breakpoints and watchers. By default it comes with support for NodeJS and can debug anything that is transpiled to JavaScript but other runtimes like C++ or Python will require an extension to be installed.**
+
+3.**There is a built in GUI for Git for the most common commands which makes it really easy to instantly see the changes you're making in your project.**
+
+4.**VS Code offers icon theming so that you can switch between icon themes in the same manner you'd switch code themes.**
+
+5.**The ability to work with a full featured integrated terminal let's you run everything you normally would from your terminal of choice.** 
+
+**Useful Plugins of VS Code**
```
Comment (VisaaliSuccessive) [inline vscode.md]: make this as h2 tag

File: vscode.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  Adds support for ESLint and should start working automatically after installing and reloading VS Code.**
+
+6.**Open-in-Browser
+  VSCode doesn't offer any built-in interface for opening files directly in the browser. This extension adds a Open With Default Browser item to the contextual menu, as well as command palette options for opening in a client of your choice (Firefox, Chrome, IE).** 
+
+7.**Faker
+  Quickly insert placeholder data using the popular Faker JavaScript library. You can generate random names, addresses, images, phone numbers, or just paragraphs of classic Lorem Ipsum.** 
+
+8.**CSS Peek
+  WIth this extension you can trace the definitions of CSS classes and ids in your stylesheets. When you right click on a selector in your HTML files, choosing the options Go to Definition and Peek definition will send you to the CSS code in which you've styled them.**
```
Comment (VisaaliSuccessive) [inline vscode.md]: Unbold these i.e normal font-weight

File: vscode.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    Git Easy: Add All Modified
+    Git Easy: Commit
+    Git Easy: Pull Current Branch from Origin
+    Git Easy: Push Current Branch to Origin
+    Git Easy: Push Current Branch (to any remote)
+    Git Easy: Status
+    Git Easy: Create New Branch
+    Git Easy: Change/Checkout Existing Branch
+    Git Easy: Log All
+    Git Easy: Log Current File**
```
Comment (VisaaliSuccessive) [inline vscode.md]: change this into GitHub code block

File: vscode.md (hunk 1 above)
Comment (ajayrose25) [reply vscode.md]: Done

File: vscode.md (hunk 2 above)
Comment (ajayrose25) [reply vscode.md]: Done

File: vscode.md (hunk 3 above)
Comment (ajayrose25) [reply vscode.md]: Done

File: vscode.md (hunk 4 above)
Comment (ajayrose25) [reply vscode.md]: Done

File: vscode.md (hunk 5 above)
Comment (ajayrose25) [reply vscode.md]: Done

**Pull request merged**